dashboard/test/models/peer_review_test.rb

```diff
@@ -59,54 +61,52 @@ class PeerReviewTest < ActiveSupport::TestCase
     @level.peer_reviewable = 'false'
     @level.save
 
-    assert_no_difference('PeerReview.count', PeerReview::REVIEWS_PER_SUBMISSION) do
+    assert_no_difference('PeerReview.count') do
       track_progress @level_source.id
     end
   end
 
-  test 'resubmitting for review should remove unassigned PeerReview objects' do
-    initial = PeerReview.count
+  test 'resubmitting for review should create new PeerReview objects' do
+    original_count = PeerReview.count
+
     track_progress @level_source.id
-    assert_equal PeerReview::REVIEWS_PER_SUBMISSION, PeerReview.count - initial
+    assert_equal 2, PeerReview.count - original_count
```

Comment:
could use `assert_difference`

Reply:
I noticed some thing werid - when I used assert difference in two places in this test, it would look for a diff of 2 the first time and 4 the second time. Not sure if that's intentional but I did this as a shortcut